<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>take</class>
 <widget class="QDialog" name="take">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>领取快递界面</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_keyboard_take">
    <item row="2" column="0">
     <widget class="QPushButton" name="pb7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pb9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pb1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pb3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbDelete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pbReturn_take">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pb5</zorder>
   <zorder>pb7</zorder>
   <zorder>pb2</zorder>
   <zorder>pb3</zorder>
   <zorder>pb8</zorder>
   <zorder>pb9</zorder>
   <zorder>pb1</zorder>
   <zorder>pb6</zorder>
   <zorder>pb4</zorder>
   <zorder>pbDelete</zorder>
   <zorder>pb0</zorder>
   <zorder>pbReturn_take</zorder>
  </widget>
  <widget class="QSplitter" name="splitter_take">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QGridLayout" name="gridLayout_take">
     <item row="1" column="0">
      <widget class="QLabel" name="label_statustitle">
       <property name="text">
        <string>领取结果</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_PassWord">
       <property name="text">
        <string>领取密码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lePassWord"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbOK">
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
